pcdsdevices/pseudopos.py

```diff
@@ -0,0 +1,83 @@
+from ophyd.device import Component as Cpt
+from ophyd.pseudopos import PseudoPositioner, PseudoSingle
+
+
+class SyncAxesBase(PseudoPositioner):
+    """
+    Synchronized Axes.
+
+    This will move all axes in a coordinated way, retaining offsets.
+
+    This can be configured to report its position as the min, max, mean, or any
+    custom function acting on a list of positions. Min is the default.
+
+    You should subclass this by adding real motors as components. The class
+    will pick them up and include them correctly into the coordinated move.
+
+    Parameters
+    ----------
+    position_mode: ``func``, optional
+        The function to apply to the list of current positions to determine
+        the combined position. By default, this is the minimum of all axis
+        positions.
+
+    Attributes
+    ----------
+    axes: ``list of str``
+        Names of the components that correspond to the real axes.
```

Comment:
You mind just adding a simple example here? I understand what you meant but I think seeing it might help newer folks:

```python
class Parallel(SyncAxesBase):
     left = Cpt(EpicsMotor, ':01')
     right = Cpt(EpicsMotor(':02')
```
?

Reply:
I agree that any of these `Base` classes should have an example, I'll add it in.

The docstring in this code snippet is wrong btw... from an old version of the file.